core/templates/pages/learner-dashboard-page/learner-dashboard-page.component.html

```
@@ -93,6 +93,21 @@
             {{ LEARNER_DASHBOARD_SECTION_I18N_IDS.MY_CERTIFICATES | translate }}
           </button>
         </li>
+        <li *ngIf="isNewExplorationEditorFeedbackTabEnabled()">
+          <button aria-label="Go to my suggestions tab"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)